compiler/msgs.nim

```diff
@@ -907,3 +960,6 @@ ropes.errorHandler = proc (err: RopesError, msg: string, useWarning: bool) =
   of rCannotOpenFile:
     rawMessage(if useWarning: warnCannotOpenFile else: errCannotOpenFile, msg)
 
+# enable colors by default on terminals
+if terminal.isatty(stdout):
```

Comment:
@Araq i don't think it is good idea, eg. when you pipe output to the file, you will be calling colors API when there's no command line window at all.